### PR TITLE
Fix: Cannot assign null to property Areablock::$currentIndex of type array

### LINKED ISSUE #15662: [Bug]: Document version can not be opened after Pimcore 11 update
### Pimcore version

v11.0.5

### Steps to reproduce

Create a Pimcore 10.6 document with a version. Update to pimcore 11. Try to preview the version of the document created in Pimcore 10.6.

### Actual Behavior

Error "Cannot assign null to property Pimcore\Model\Document\Editable\Areablock::$currentIndex of type array" thrown
Cannot assign null to property version error

### Expected Behavior

Display the document in the proper version

### PR DESCRIPTION
**Pull request by blankse**
Fix: https://github.com/pimcore/pimcore/issues/15662

### CODE REVIEW

File: models/Document/Editable/Areablock.php [hunk 1]
```diff
@@ -51,16 +51,16 @@ class Areablock extends Model\Document\Editable implements BlockInterface
...
-    protected array $currentIndex;
+    protected ?array $currentIndex = null;
 
     /**
      * @internal
      *
      * @var bool
      */
-    protected bool $blockStarted;
+    protected bool $blockStarted = false;
```
Comment (Cruiser13) [inline models/Document/Editable/Areablock.php]: This should also be optional to prevent further errors:
`protected ?bool $blockStarted = false;`

**Review by Cruiser13 — COMMENTED** [review]
$blockStarted should also be optional

File: models/Document/Editable/Areablock.php (hunk 1 above)
Comment (blankse) [reply models/Document/Editable/Areablock.php]: @Cruiser13 Done https://github.com/pimcore/pimcore/pull/15702/commits/94983b28a6dc6a71d9397559776764a5b313bc9a

**Comment by dvesh3** [comment]
@blankse thanks!

**Pull request merged**